README.md

```diff
@@ -51,17 +51,43 @@ a double hashtag like so:
 
 Lines with only a single `#` will not appear as section in `task:help` and can be seen as plain comments.
 
-### Help command
+## Help command
 
 Running `./Taskfile help`, the `task:help` function is triggered. This task will list all available sections and tasks
 using the double `##` comments you've learned about above. Now it's clear how you can run any other task!
 
-## Credits
+# Auto-completion
+
+Autocompletion works when you use `zsh` and `oh-my-zsh`. Create the following file in your oh-my-zsh directory
+`~/.oh-my-zsh/completions/_task.zsh`:
+
+```shell
+#compdef task
+
+_task() {
+	local -a commands
```

Comment:
What is `-a`, is there a written out variant available?

Reply:
No, Bash does not provide a long-form alternative flag for `-a` when declaring an indexed array with local. The `-a` option is the only way to specify that a variable should be an indexed array.